### DISCUSSION TITLE
Modern MoGal if - Oni's Love Rule - SuicaoDex

### DISCUSSION THREAD

**Discussion by giscus** [TNTKien/suicaodex#11] [Manga]
# Modern MoGal if - Oni's Love Rule - SuicaoDex

This "if" series is a set of spin-off stories that take place in the Modern Mogal world drawn by Bliss.  
  
Original comic by [Shepherd0821](https://mangadex.org/group/6025/shepherd0821), uploaded to MangaDex with permission.

https://suicaodex.com/manga/36222216-195e-41bb-a3e6-a2c8d8298a07

**Comment by dex593** [comment]
truyện tạm được
